libmachine/host/host.go

```diff
@@ -149,6 +149,10 @@ func (h *Host) GetURL() (string, error) {
 	return h.Driver.GetURL()
 }
 
+func (h *Host) GetDockerVersion() (string, error) {
```

Comment:
Nitpick: The idiomatic Golang convention would be to call this function simply `DockerVersion`.